network-api/networkapi/templates/pages/base.html

```diff
@@ -40,8 +41,13 @@
       {% block heroGuts %}{% endblock %}
       {% block content %}{% endblock %}
 
+      {% block footer_block %}
```

Comment:
these blocks allow us to rely on base.html without being specifically tied to the foundation site for things like navs/footers/etc